android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerDelegate.java

```diff
@@ -85,6 +85,8 @@ public boolean onActivityResult(final int requestCode, final int resultCode, fin
         // Save file
         if (requestCode == SAVE_FILE_CODE) {
             if (resultCode == Activity.RESULT_OK) {
+                if (data == null)
```

Comment:
Since the other if's also use curly braces, can you do that here as well?

Also, if you update the patch version and add an entry in the changelog, then this can be published.

Reply:
Sure, I added all of this